mkosi/__init__.py

```diff
@@ -6623,33 +6634,48 @@ def interface_exists(dev: str) -> bool:
 
 def find_address(args: CommandLineArguments) -> Tuple[str, str]:
     name = virt_name(args)
+    last_exception = None
```

Comment:
This line is not strictly necessary here, isn't it? It will be reset right at the start of the loop.

Reply:
Ah, quite indeed!